doc/classes/Control.xml

```diff
@@ -12,7 +12,7 @@
 		Call [method accept_event] so no other node receives the event. Once you accept an input, it becomes handled so [method Node._unhandled_input] will not process it.
 		Only one [Control] node can be in focus. Only the node in focus will receive events. To get the focus, call [method grab_focus]. [Control] nodes lose focus when another node grabs it, or if you hide the node in focus.
 		Sets [member mouse_filter] to [constant MOUSE_FILTER_IGNORE] to tell a [Control] node to ignore mouse or touch events. You'll need it if you place an icon on top of a button.
-		[Theme] resources change the Control's appearance. If you change the [Theme] on a [Control] node, it affects all of its children. To override some of the theme's parameters, call one of the [code]add_theme_*_override[/code] methods, like [method add_theme_font_override]. You can override the theme with the Inspector.
+		[Theme] resources change the [Control]'s appearance. The [member theme] of a [Control] node affects all of its direct and indirect children (as long as a chain of [Control]s is uninterrupted). To override some of the theme items, call one of the [code]add_theme_*_override[/code] methods, like [method add_theme_font_override].
```

Comment:
I think it's worth lightly mentioning they can be overridden in the Inspector dock, again.

Reply:
Should I restore the original phrasing " You can override the theme with the Inspector.", then?
Or were you thinking incorporate it into another of the sentences

Reply:
Something like "It's also possible to override theme items in the Inspector." or "You can override theme items in the Inspector". The prior sentence is exactly about this, and it's very much for granted that `theme` can be changed in the Inspector like most properties.

Reply:
I went with the second suggestion.